.travis.yml

```diff
@@ -56,15 +56,15 @@ script:
              -DENABLE_PYTHON=ON
              -DINSTALL_CWRAP=OFF
              -DBUILD_APPLICATIONS=ON
-             -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR
              -DINSTALL_ERT_LEGACY=ON
-             -DCMAKE_PREFIX_PATH=$INSTALL_DIR
              -DERT_USE_OPENMP=ON
              -DCMAKE_C_FLAGS='-Werror=all'
              -DCMAKE_CXX_FLAGS='-Werror -Wno-unused-result'
   - make
-  - make install
-  - export PYTHONPATH=$INSTALL_DIR/lib/python2.7/dist-packages:$INSTALL_DIR/lib64/python2.7/dist-packages:$PYTHONPATH
-  - export PYTHONPATH=$INSTALL_DIR/lib/python2.7/site-packages:$INSTALL_DIR/lib64/python2.7/site-packages:$PYTHONPATH
+  - sudo make install
```

Comment:
Would it now work if you reinsert the `INSTALL_PREFIX` passed to cmake and then remove the `sudo`?

Reply:
It would, but this is definitely space consuming. We would need to update `PYTHONPATH` anyway: it would contain `INSTALL_PREFIX` and `/usr/lib/...`, and we would need to include these lines also into cmake.

Reply:
Do as you please :+1: